.github/workflows/test-android.yml

```
@@ -0,0 +1,130 @@
+name: End To End Tests
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - 'CODE_OF_CONDUCT.md'
+      - 'CONTRIBUTING.md'
+      - 'LICENSE'
+      - 'SECURITY.md'
+      - 'docs/**'
+      - '.github/**'
+      - '!.github/workflows/test.yml'
+  push:
+    branches:
+      - develop
+      - main
+      - release/*
+    tags: '[0-9]+.[0-9]+.[0-9]+'
+    paths-ignore:
+      - 'README.md'
+      - 'CODE_OF_CONDUCT.md'
+      - 'CONTRIBUTING.md'
+      - 'LICENSE'
+      - 'SECURITY.md'
+      - 'docs/**'
+      - '.github/**'
+      - '!.github/workflows/test.yml'
```

This should be the current workflow file
```suggestion
      - '!.github/workflows/test-android.yml'
```

---

Done